chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/dao/ChangeRequestDAO.java

```diff
@@ -206,6 +222,39 @@ private List<ChangeRequestEntity> getEntitiesByDevelopers(List<Long> developerId
         return results;
     }
 
+    private List<ChangeRequestEntity> getSearchResultEntitiesByTypeAndPeriod(Long crTypeId, Long periodId) {
+        String hql = "SELECT DISTINCT cr "
+                + "FROM ChangeRequestEntity cr "
+                + "JOIN FETCH cr.changeRequestType crType "
+                + "JOIN FETCH cr.developer dev "
+                + "LEFT JOIN FETCH dev.address "
+                + "LEFT JOIN FETCH dev.contact "
+                + "LEFT JOIN FETCH dev.statusEvents statusEvents "
+                + "LEFT JOIN FETCH statusEvents.developerStatus "
+                + "LEFT JOIN FETCH dev.attestations devAtt "
+                + "LEFT JOIN FETCH devAtt.period per "
+                + "LEFT JOIN FETCH dev.certificationBodyMaps devAcbMaps "
+                + "LEFT JOIN FETCH devAcbMaps.certificationBody devAcb "
+                + "LEFT JOIN FETCH devAcb.address "
```

Comment:
I think this is going to cause a merge conflict with OCD-3940.  We are moving the acb relation to directly on CR, rather than going through the developer object.

Reply:
Added comment as discussed